### ISSUE TITLE
Make assignme run at system startup

### ISSUE THREAD

**Issue by kudapara** [kudapara/assignme#4]
Currently when using assignme you have to manually open it every time you want to use it. A nice to have feature would be to have it run at startup and show you a notification with some pending tasks.

**Comment by Chandrasekar-G** [comment]
I'm on it

**Comment by aflmp** [comment]
I can see that end result is a system application, in my case when i build, i get dmg file, which on installation works fine. A user can go to settings and this application to login startup programs, if they want to.
Is there something that i'm missing here?

**Comment by kudapara** [comment]
That's all correct, but I think that if the application can show some push notifications when it is executed on system startup it could help the user to take action on the pending tasks. Sometimes I even forget to run the app to check for pending tasks and when I finally do some of them will be overdue 😞. So I am hoping that some notification mechanism of some sort could solve the problem.
